coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/TrinoSqlDialect.java

```diff
@@ -31,8 +31,36 @@ public boolean supportsCharSet() {
   }
 
   /**
-   * Override this method so that there will be explicit and correct table alias for selected fields, which is necessary for
-   * data type derivation on SqlNodes
+   * Override this method so that there will be explicit table alias for all the fields, which is necessary for
+   * data type derivation on struct fields.
+   *
+   * For the following input SQL:
+   * CREATE TABLE default.complex(s struct(name:string, age:int));
```

Comment:
This comment and the override itself are going away in the future, correct?

Reply:
Yes